.github/workflows/docReport.yml

```diff
@@ -51,14 +51,14 @@ jobs:
         if: ${{ github.event_name == 'pull_request' }}
         env:
           DOCS_REPORT_TOKEN: ${{ secrets.DOCS_REPORT_DATASET_TOKEN }}
-          DOCS_REPORT_DATASET: ${{ github.ref_name }}
+          DOCS_REPORT_DATASET: ${{ github.event.number }}
```

Comment:
should we prefx it so it's easy to identify as a temporary dataset?

Reply:
sure I can take a look at how to do that